frontend/app/upload/components/FileUploader/hooks/useFileUploader.ts

```diff
@@ -38,7 +38,7 @@ export const useFileUploader = () => {
         });
       }
     },
-    [session.access_token, publish]
```

Comment:
Do we want to remove the sesson.access_token from the dependencies ?

Reply:
What linter are you using ?

Reply:
I think it was how I was running the project previously not in the container. It then used my default eslint settings. It's fine now. 2 mins and ill remove that.

Reply:
The devx is pretty bad with a container on the frontend, so you'll probably run the project outside of it again. Maybe you should change your settings or the project's to share the same

Reply:
Yep agreed. I really dont like the container :)